Vladyslav-Hudz/README.md

```diff
@@ -38,6 +33,11 @@ Before getting started, you will need:
 - Search - search for keywords in your code.
 - Open Serial Monitor - opens the Serial Monitor tool, as a new tab in the console.
 
+## Step 2: Connecting Arduino
+
+1. Launch the Arduino IDE.
+2. Connect Arduino to your computer using the USB cable.
+
 ## Step 3: Selecting the board and port
 
 1. In the `Menu Bar` select `Tools` > `Board` > `Arduino AVR Boards`.
```

Comment:
Add a comma after the Menu Bar and no need to capitalize it --> In the menu bar, select Tools ...